lib/msal-common/src/auth/MsalAccount.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ClientInfo } from "./ClientInfo";
+import { IdToken } from "./IdToken";
+import { StringUtils } from "../utils/StringUtils";
+import { StringDict } from "../utils/MsalTypes";
+import { ICrypto } from "../utils/crypto/ICrypto";
+
+/**
+ * accountIdentifier       combination of idToken.uid and idToken.utid
+ * homeAccountIdentifier   combination of clientInfo.uid and clientInfo.utid
+ * userName                idToken.preferred_username
+ * name                    idToken.name
+ * idToken                 idToken
+ * sid                     idToken.sid - session identifier
+ * environment             idtoken.issuer (the authority that issues the token)
+ */
+export class MsalAccount {
```

Comment:
For my own understanding, this is just basically just an abstraction around an id token, right? This is effectively our idea of some generic account and the information surrounding it?

Reply:
It's a combination of id token and client info, which is why we persist those two after login is completed.